app/views/includes/contact.jade

```diff
@@ -0,0 +1,22 @@
+//- CONTACT ::
+section#contact
+    .contact-title.left
+        h4.titulo #{contact.contact_title}
+        span.bold.white
+            p.ct-t1 #{contact.contact_subtitle}
+            p.ct-t2 #{contact.call_to_action}
+    .contact-form
+        form#contactForm.ct-form(action='mailto:info@upplication.com')
```

Comment:
name id must not be "camel case", we are using "snake case": https://en.wikipedia.org/wiki/Snake_case